server/routes/authentication.js

```diff
@@ -24,29 +24,39 @@ router.post("/api/authentication/google", async (req, res) => {
       res.status(500).send("Authentication error.");
       return;
     }
+    const accessToken = {
+      token: token.access_token,
+      exp: token.expiry_date,
+    };
+    const tokens = {
+      id_token: token.id_token,
```

Comment:
Is there a reason these have to be snakecase, or can we make them camelcase?

Reply:
It's probably old code we overlooked. I should fix it in a different PR in case the variable is used elsewhere I'm not aware of.